src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/CompositeMLDsa/CompositeMLDsaImplementationTests.cs

```diff
@@ -39,6 +41,40 @@ private static void AssertCompositeMLDsaIsOnlyPublicAncestor(Func<CompositeMLDsa
             Assert.Equal(typeof(CompositeMLDsa), keyType);
         }
 
+        [Fact]
+        public static void UseAfterDispose()
+        {
+            CompositeMLDsa dsa = CompositeMLDsa.GenerateKey(CompositeMLDsaAlgorithm.MLDsa65WithECDsaP384);
+            dsa.Dispose();
+            dsa.Dispose(); // no throw
+
+            CompositeMLDsaTestHelpers.VerifyDisposed(dsa);
+        }
+
+        [Fact]
+        public static void ImportPkcs8_BerEncoding()
+        {
+            // Pkcs8 is DER encoded, so create a BER encoding from it by making it use a non-minimal length encoding.
+            byte[] key = CompositeMLDsaTestData.AllIetfVectors[0].Pkcs8;
```

Comment:
This is the only reference I can find to any of the Pkcs8 data already in the TestData file.  I find that a little surprising.

Maybe "foreach vector, import p8 and import spk8, sign with private, verify with public" is too much, but AFAICT, this BER-transformed input is the only PKCS8-based known-answer-test.

Reply:
Added roundtrip tests for SPKI and PKCS8.